beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,491 @@ title: Quick Start
+React will call your component function again. This time, `count` will be `1`. Then it will be `2`. And so on.
 
-For example, this form has a **redundant** `fullName` state variable:
+If you render the same component multiple times, each will get its own state. Try clicking each button separately:
 
 <Sandpack>
 
 ```js
 import { useState } from 'react';
 
-export default function Form() {
-  const [firstName, setFirstName] = useState('');
-  const [lastName, setLastName] = useState('');
-  const [fullName, setFullName] = useState('');
+function MyButton() {
+  const [count, setCount] = useState(0);
 
-  function handleFirstNameChange(e) {
-    setFirstName(e.target.value);
-    setFullName(e.target.value + ' ' + lastName);
+  function handleClick() {
+    setCount(count + 1);
   }
 
-  function handleLastNameChange(e) {
-    setLastName(e.target.value);
-    setFullName(firstName + ' ' + e.target.value);
-  }
+  return (
+    <button onClick={handleClick}>
+      Clicked {count} times
+    </button>
+  );
+}
 
+export default function MyApp() {
   return (
-    <>
-      <h2>Let’s check you in</h2>
-      <label>
-        First name:{' '}
-        <input
-          value={firstName}
-          onChange={handleFirstNameChange}
-        />
-      </label>
-      <label>
-        Last name:{' '}
-        <input
-          value={lastName}
-          onChange={handleLastNameChange}
-        />
-      </label>
-      <p>
-        Your ticket will be issued to: <b>{fullName}</b>
-      </p>
-    </>
+    <div>
+      <h1>Counters that update separately</h1>
+      <MyButton />
+      <MyButton />
+      <MyButton />
+    </div>
   );
 }
 ```
 
 ```css
-label { display: block; margin-bottom: 5px; }
+button {
+  display: block;
+  margin-bottom: 5px;
+}
 ```
 
 </Sandpack>
 
-You can remove it and simplify the code by calculating `fullName` while the component is rendering:
+Functions starting with `use` are called Hooks. `useState` is a built-in Hook provided by React. You can find other built-in Hooks in the [React API reference](/reference). You can also write your own Hooks by combining the existing ones.
 
-<Sandpack>
+Hooks are more restrictive than regular functions. You can only call Hooks *at the top level* of your components (or other Hooks). If you want to `useState` in a condition or a loop, extract a new component and put it there.
 
-```js
-import { useState } from 'react';
+## Sharing data between components {/*sharing-data-between-components*/}
 
-export default function Form() {
-  const [firstName, setFirstName] = useState('');
-  const [lastName, setLastName] = useState('');
+In the above example, each counter is independent. If you want them to display the same value and update together, *move the state up* to their closest common ancestor. First, move state from `MyButton` into `MyApp`:
```

Comment:
**problem**: Since it's a new section, it feels like the "above example" is missing

**possible solution**: Replace "In the above example" with "In the previous section's example" or "In the previous example"